tests/ui/type/typeid-consistency-comprehensive.rs

```diff
@@ -1,16 +1,18 @@
+//! Comprehensively checks the correctness and consistency of `std::any::TypeId::of`.
```

Comment:
```suggestion
//! Checks the correctness and consistency of `std::any::TypeId::of`.
```

No test for `TypeId` will ever be comprehensive, there's always more 😆 please rename the test file too

Reply:
Oh, also explainable, I saw a big test = comprehensive :D